demos/OH-EXF-Kamakura/src/app-camera-management/service.policy.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Need to add the license header as I remember you adding for other json files before.